docs/orchestration.md

```diff
@@ -278,6 +277,61 @@ Even though **stack-c** defined that it needs to be run after **stack-b**, since
 **runtime** order.
 
 
+## Execution Environment
+
+The environment in which stacks are going to be executed can be configured
+using the `terramate.config.run.env` block. This block has no labels and
+accepts arbitrary attributes where each attribute represents an environment
+variable that will be exported when executing the stack.
```

Comment:
```suggestion
variable that will be evaluated and exported when executing the stack.
```

so users know they can also create environment variables referencing `terramate.stack.*` variables.

Reply:
one example would be:

```
terramate {
  config {
    run {
      env {
        PATH = "${terramate.stack.path.absolute}/bin}:${env.PATH}"
      }
    }
  }
}
```